field-iteration.go

```diff
@@ -63,7 +63,7 @@ func (iteration *FieldIteration) SQLOptions() (*sql.Options, error) {
 		result.Name = snakecase.SnakeCase(iteration.Name())
 	}
 
-	if len(result.Type) == 0 {
+	if len(result.Type) == 0 && !result.IsInline {
```

Comment:
could you explain a bit what's happening here and why we don't proceed if it's IsInline? are there any other cases where we want to do things differently when we  have IsInline?